torch/csrc/jit/runtime/decomposition_registry.cpp

```diff
@@ -108,5 +100,32 @@ void RunDecompositions(std::shared_ptr<Graph> g) {
   }
 }
 
+c10::optional<std::shared_ptr<Graph>> DecompositionGraphForSchema(
+    const FunctionSchema& schema) {
+  loadDecompositionFunctions();
+  GRAPH_DEBUG("Trying to find schema: ", schema);
+  auto cache_it = schema_to_decomposition.find(&schema);
+  if (cache_it != schema_to_decomposition.end()) {
+    return cache_it->second;
+  }
+  GRAPH_DEBUG("Could not find schema: ", schema);
+
+  return c10::nullopt;
+}
+
+c10::optional<GraphFunction*> GetDecompositionFunction(
+    const FunctionSchema& schema) {
+  loadDecompositionFunctions();
+  auto cache_it = schema_to_function.find(&schema);
+  GRAPH_DEBUG("Trying to find schema: ", schema);
+  if (cache_it == schema_to_function.end()) {
+    GRAPH_DEBUG("Could not find schema: ", schema);
+    return c10::nullopt;
+  }
```

Comment:
nit: could you call DecompositionGraphForSchema() here to avoid duplicating this code?